src/main/flight/gps_rescue.c

```diff
@@ -650,8 +667,11 @@ void descend(void)
         // if quad drifts further than 2m away from home, should by then have rotated towards home, so pitch is allowed
         rescueState.intent.rollAngleLimitDeg = gpsRescueConfig()->maxRescueAngle * rescueState.intent.proximityToLandingArea;
         // reduce roll capability when closer to home, none within final 2m
+        rescueState.intent.pitchAngleLimitDeg = gpsRescueConfig()->maxRescueAngle / rescueState.sensor.groundspeedPitchAttenuator;
```

Comment:
```suggestion
        rescueState.intent.pitchAngleLimitDeg = gpsRescueConfig()->maxRescueAngle * rescueState.sensor.groundspeedPitchAttenuator;
```
It will save one divide and IMO is easier to understand

Reply:
Yes, thank you, it is definitely better like this.  Very grateful for your suggestions!